cmd/state/internal/cmdtree/hello_example.go

```diff
@@ -0,0 +1,75 @@
+package cmdtree
+
+import (
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/hello"
+)
+
+func newHelloCommand(prime *primer.Values) *captain.Command {
+	runner := hello.New(prime)
+
+	params := hello.NewRunParams()
+
+	cmd := captain.NewCommand(
+		"hello",
+		locale.Tl("hello_cmd_title", "Saying hello"),
+		locale.Tl("hello_cmd_description", "An example command"),
+		prime,
+		[]*captain.Flag{},
```

Comment:
Can you add an example flag and add a command similar to the command name on saying it should not be localized?

Reply:
Maybe a good opportunity to showcase custom flag types..